src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/LaboratoryInvestigation.java

```diff
@@ -11,7 +11,13 @@
 import java.util.Optional;
 
 /**
- * Example: INV+MQ+42R4.:911::Serum ferritin'
+ * Details of the laboratory investigation.
+ * <pre>
+     * Example in a Pathology (NHS003) message: INV+MQ+42R4.:911::Serum ferritin'
```

Comment:
Looks like IntelliJ has been "helpful" again. Also suggest wrapping the examples in `{@code content}`
```suggestion
 * Example in a Pathology (NHS003) message: {@code INV+MQ+42R4.:911::Serum ferritin'}
```